src/op_mode/show_lacp.py

```diff
@@ -0,0 +1,101 @@
+#
+# This script will parse 'sudo cat /proc/net/bonding/<interface name>' and output a tables for 'lacp neighbors' and 'lacp details'
+# lacp neighbors shows - "Interface, Members, Mode, Rate, System Mac, Minimum Links, Hash"
+# lacp details shows   - "Interface, Members, Mode, Rate, System MAC, Hash"
+
+from tabulate import tabulate
+import subprocess
+import re
+import argparse
+
+def get_all_bonds():
+    return subprocess.run(f"cli-shell-api listActiveNodes interfaces bonding", stdout=subprocess.PIPE, shell=True, text=True).stdout.encode('utf-8').decode('utf-8')
+
+def get_lacp_neighbors(interface):
+    headers = ["Interface", "Member", "Local ID", "Remote ID"]
+    data = subprocess.run(f"sudo cat /proc/net/bonding/{interface}", stdout=subprocess.PIPE, stderr=subprocess.DEVNULL, shell=True, text=False).stdout.decode('utf-8')
+    if 'Bonding Mode: IEEE 802.3ad Dynamic link aggregation' not in data:
+        exit("Bond is not configured with mode 802.3ad")
+
+    pattern = re.compile(
+        r"Slave Interface: (?P<member>\w+\d+).*?"
+        r"system mac address: (?P<local_id>[0-9a-f:]+).*?"
+        r"details partner lacp pdu:.*?"
+        r"system mac address: (?P<remote_id>[0-9a-f:]+)",
+        re.DOTALL
+    )
+
+    interfaces = []
+
+    for match in re.finditer(pattern, data):
+        member = match.group("member")
+        local_id = match.group("local_id")
+        remote_id = match.group("remote_id")
+        interfaces.append([interface, member, local_id, remote_id])
+
+    if interfaces:
+        print(tabulate(interfaces, headers))
+    else:
+        print("No Member interfaces found!")
+
+def get_bond_info(interface):
+    headers = ["Interface", "Members", "Mode", "Rate", "System MAC", "Hash"]
+    data = subprocess.run(f"sudo cat /proc/net/bonding/{interface}", stdout=subprocess.PIPE, stderr=subprocess.DEVNULL, shell=True, text=False).stdout.decode('utf-8')
+    if 'Bonding Mode: IEEE 802.3ad Dynamic link aggregation' not in data:
+        if args.detailall:
+            return None, headers
+        else:
+            exit("Bond is not configured with mode 802.3ad")
+
+    mode_active = "active" if "LACP active: on" in data else "passive"
+    lacp_rate = re.search(r"LACP rate: (\w+)", data).group(1) if re.search(r"LACP rate: (\w+)", data) else "N/A"
+    hash_policy = re.search(r"Transmit Hash Policy: (.+?) \(\d+\)", data).group(1) if re.search(r"Transmit Hash Policy: (.+?) \(\d+\)", data) else "N/A"
+    system_mac = re.search(r"System MAC address: ([0-9a-f:]+)", data).group(1) if re.search(r"System MAC address: ([0-9a-f:]+)", data) else "N/A"
+
+    members = ",".join(set(re.findall(r"Slave Interface: ([a-zA-Z0-9:_-]+)", data)))
+
+    table_data = []
+
+    table_data = [interface, members, mode_active, lacp_rate, system_mac, hash_policy]
+
+    if table_data:
+        return table_data, headers
+    else:
+        print("No member interfaces found!")
+
+if __name__ == "__main__":
```

Comment:
All new operational mode scripts must use the new approavh with `vyos.opmode` so that they can be automatically exposed in the API as well.
See https://blog.vyos.io/writing-vyos-operational-mode-commands-just-got-better and a new-style script like https://github.com/vyos/vyos-1x/blob/current/src/op_mode/cpu.py

Reply:
Is there a specific format the dictionary output needs to be?

Reply:
Is something like this sufficient:
```
{
    "lacp": [
        {
            "interface": "bond0",
            "members": "eth3",
            "mode": "active",
            "rate": "slow",
            "system_mac": "aa:bb:cc:dd:ee:f0",
            "hash": "layer2"
        },
        {
            "interface": "bond1",
            "members": "",
            "mode": "active",
            "rate": "slow",
            "system_mac": "aa:bb:cc:dd:ee:f0",
            "hash": "layer2"
        }
    ]
}
```

Reply:
@dmbaturin I updated the script to use vyos.opmode